modules/logging-forward-splunk.adoc

```diff
@@ -31,34 +30,38 @@ $ oc -n openshift-logging create secret generic vector-splunk-secret --from-lite
 +
 [source,yaml]
 ----
-apiVersion: logging.openshift.io/v1
+apiVersion: observability.openshift.io/v1
 kind: ClusterLogForwarder
 metadata:
-  name: <log_forwarder_name> <1>
-  namespace: <log_forwarder_namespace> <2>
+  name: <log_forwarder_name>
+  namespace: openshift-logging
 spec:
-  serviceAccountName: <service_account_name> <3>
+  managementState: Managed
+  serviceAccountName: <service_account_name> #<1>
   outputs:
-    - name: splunk-receiver <4>
-      secret:
-        name: vector-splunk-secret <5>
-      type: splunk <6>
-      url: <http://your.splunk.hec.url:8088> <7>
-  pipelines: <8>
-    - inputRefs:
+    - name: splunk-receiver #<2>
+      type: splunk #<3>
+      splunk:
+        authentication:
+          token:
+            key: hecToken # <4>
+            secretName: splunk-secret
+        url: '<http://your.splunk.hec.url:8088>' #<5>
+        index: '{.log_type||"undefined"}' #<6>
```

Comment:
This is optional, and so are several other fields.    Do you want to include them as per the upstream [example](https://github.com/openshift/cluster-logging-operator/blob/master/docs/features/logforwarding/outputs/splunk-forwarding.adoc#steps-to-forward-to-splunk-http-event-collector-hec)?